modules/integration/tests-platform/tests-clustering/src/test/java/org/wso2/mb/platform/tests/clustering/slot/SlotDeliveringTestCase.java

```diff
@@ -0,0 +1,158 @@
+import org.wso2.mb.integration.common.clients.configurations.AndesJMSConsumerClientConfiguration;
+import org.wso2.mb.integration.common.clients.configurations.AndesJMSPublisherClientConfiguration;
+import org.wso2.mb.integration.common.clients.operations.clients.AndesAdminClient;
+import org.wso2.mb.integration.common.clients.operations.utils.AndesClientConstants;
+import org.wso2.mb.integration.common.clients.operations.utils.AndesClientUtils;
+import org.wso2.mb.integration.common.clients.operations.utils.ExchangeType;
+import org.wso2.mb.platform.common.utils.MBPlatformBaseTest;
+import org.xml.sax.SAXException;
+
+import javax.xml.stream.XMLStreamException;
+import javax.xml.xpath.XPathExpressionException;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.rmi.RemoteException;
+
+import static org.testng.Assert.assertTrue;
+
+public class SlotDeliveringTestCase extends MBPlatformBaseTest {
+
+	/**
+	 * Prepare environment for test
+	 * @throws LoginAuthenticationExceptionException
+	 * @throws IOException
+	 * @throws XPathExpressionException
+	 * @throws URISyntaxException
+	 * @throws SAXException
+	 * @throws XMLStreamException
+	 */
+	@BeforeClass(alwaysRun = true)
+	public void init()
+			throws LoginAuthenticationExceptionException, IOException, XPathExpressionException,
+			       URISyntaxException, SAXException, XMLStreamException, AndesAdminServiceBrokerManagerAdminException {
+		super.initCluster(TestUserMode.SUPER_TENANT_ADMIN);
+		super.initAndesAdminClients();
+
+		String randomInstanceKey = getRandomMBInstance();
+
+		AndesAdminClient tempAndesAdminClient = getAndesAdminClientWithKey(randomInstanceKey);
+
+		if (tempAndesAdminClient.getQueueByName("clusterSingleQueue1") == null) {
+			tempAndesAdminClient.createQueue("clusterSingleQueue1");
+		}
+
+		if (tempAndesAdminClient.getQueueByName("clusterSingleQueue2") == null) {
+			tempAndesAdminClient.createQueue("clusterSingleQueue2");
+		}
+
+		if (tempAndesAdminClient.getQueueByName("clusterSingleQueue3") == null) {
+			tempAndesAdminClient.createQueue("clusterSingleQueue3");
+		}
+
+	}
+
+	/**
+	 *
+	 * @throws Exception
+	 */
+	@Test(groups = "wso2.mb", description = "Single queue Single node send-receive test case")
+	public void testSingleQueueSingleNodeSendReceive() throws Exception {
+
+		long sendCount = 1000L;
+		long expectedCount = 1000L;
+
+
+		String randomInstanceKey = getRandomMBInstance();
+
+		AutomationContext tempContext = getAutomationContextWithKey(randomInstanceKey);
+
+		AndesJMSConsumerClientConfiguration consumerConfig = new AndesJMSConsumerClientConfiguration(tempContext.getInstance().getHosts().get("default"),
+		                                                                                             Integer.parseInt(tempContext.getInstance().getPorts().get("amqp")),
+		                                                                                             ExchangeType.QUEUE, "clusterSingleQueue1");
```

Comment:
Check if this line exceeds the right margin, if so wrap. Check other lines as well.